test/api/settings_test.rb

```diff
@@ -0,0 +1,68 @@
+require 'test_helper'
+require 'json'
+
+class SettingTest < ActiveSupport::TestCase
+    include Rack::Test::Methods
+    include TestHelpers::AuthHelper
+    include TestHelpers::JsonHelper
+
+    def app
+        Rails.application
+    end
+
+    # Get config details
+    def test_get_config_details        
+        expected_ProductName =  Doubtfire::Application.config.institution[:product_name]
```

Comment:
Variable name case! This is very sloppy.

Reply:
Hi Andrew, I have modified variables name case, and unified their format.